website/content/docs/concepts/filtering/audit.mdx

```diff
@@ -1,165 +1,160 @@
-message will always be present in `fallback` devices (described below).
-
-The properties of the test message are as follows:
-
-* `mount_point` - not present
-* `mount_type` - not present.
-* `namespace` - not present.
-* `operation` - `update`.
-* `path` - `sys/audit/test`.
-
-Vault Operators should use this when trying to understand when they should or
-should not expect a test message to appear in the sink for a newly enabled audit device.
-
-## Fallback device
-
-The ability to filter audit entries can provide great flexibility to your workflows,
-however the additional complexity can make it possible for a Vault Operator to configure their
-audit devices in such a way that some audit entries are missed out from audit logs entirely.
-Therefore, we strongly encourage you to test out audit configurations in your non-production
-environments before deploying them to production.
-
-The `fallback` audit device is the (non-mandatory) mechanism by which Vault can
-continue to adhere to Vault's [security model](/vault/docs/internals/security),
-that all requests and responses are successfully logged before the client receives
-any secret material.
-
-When exclusively using filtered audit devices, enable the fallback audit device to
-catch any audit entries that would otherwise be missed.
-
-This means Vault is able to provide the same guarantee that 'at least one device must
-successfully write an audit entry' as when only standard/non-filtered audit devices
-are enabled.
-
-<Note title="Single fallback device">
-  Vault supports enabling only a single fallback audit device.
-</Note>
-
-#### Enabling the fallback audit device
-
-[Enabling the fallback audit device](/vault/docs/audit#fallback) requires supplying the `fallback` option:
-
-```shell-session
-vault audit enable -path=my-fallback -description="fallback device" file file_path=/tmp/audit.fallback.log fallback=true
-```
-
-### Metrics
-
-When the fallback device is enabled, and required to persist an audit entry to the audit log,
-Vault will emit a [fallback 'success' metric](/vault/docs/internals/telemetry/metrics/audit#vault-audit-fallback-success) on
-a successful write to the audit log.
-
-If audit devices are enabled that make use of filtering, but no fallback audit device
-has been enabled, Vault will produce a [fallback 'miss' metric](/vault/docs/internals/telemetry/metrics/audit#vault-audit-fallback-miss)
-as a way to allow Operators to understand how many auditable Vault entries are not
-being persisted to their audit logs.
+  ```
+1. Write secret data to the `kv` engine:
+  ```shell-session
+    vault kv put my-kv my_secret=always_angry
+  ```
+
+The `/var/kv-audit.log` now includes two entries: one for the command that
+enabled `my-kv` and one for the command that wrote a secret to `my-kv`. Entries
+for the other commands are captured by the fallback device. And the original
+audit file, `vault-audit.log` continues to capture all audit events.
```

Comment:
There are 4 entries as there's a request and a response for each action (sorry).

```suggestion
The `/var/kv-audit.log` now includes four entries in total: a request and response
entry for the command that enabled `my-kv` and another request and response entry
for the command that wrote a secret to `my-kv`. Entries for the other commands
are captured by the fallback device. And the original audit file, `vault-audit.log`
continues to capture all audit events.
```